### PR TITLE
Automated tests

### PR DESCRIPTION
**Pull request by mbodeantor**
Run API checks every 15 min

### CODE REVIEW

**Comment by josh-chamberlain** [comment]
nice. when these run, where would we see the test results? the `actions` section of the repo?

**Comment by mbodeantor** [comment]
@josh-chamberlain yeah, once I'm sure they work, I want to add the discord webhook too for fails

**Review by josh-chamberlain — CHANGES_REQUESTED** [review]
looks good aside from those 2 lil changes

File: regular_api_checks.py [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+        "https://data-sources.pdap.io/api/user",
+        headers=HEADERS,
+        json={"email": "test2", "password": "test"},
+    )
+
+    return response
+
+
+# archives
+def test_get_archives():
```
[the remark this answers is not in the document]
Comment (mbodeantor) [reply regular_api_checks.py:139]: Adding docs: https://app.gitbook.com/o/-MXypK5ySzExtEzQU6se/s/-MXyolqTg_voOhFyAcr-/~/changes/454/api/endpoints/data-sources-database

**Pull request merged**